tests/e2e/test_api.py

```diff
@@ -426,6 +426,15 @@ def test_model_provider():
     }
 
 
+def test_max_one_default_model_provider():
+    """Check if max. one model and provider is selected as default."""
+    states = metrics_utils.get_enable_status_for_all_models(metrics_client)
+    enabled_states = [state for state in states if state is True]
+    assert (
+        len(enabled_states) <= 1
```

Comment:
we should expect exactly one.  We control the configuration when we are testing.

Reply:
updated